app/src/features/observations/BlockObservationPage.tsx

```diff
@@ -126,7 +126,7 @@ const BlockObservationPage = () => {
       revision_count: observationWithDetailsResponse.revision_count
     });
     setTableData(observationWithDetailsResponse.data.tableData.data);
-  }, [biohubApi.observation, urlParams]);
+  }, [biohubApi.observation, observationId, projectId, surveyId]);
```

Comment:
are we sure this doesnt cause unexpected rerenders?

Reply:
added urlparams back, didn't notice issues with the others but will keep an eye.